machines/Makefile

```diff
@@ -1,50 +1,44 @@
-find-dir-with = $(shell /usr/bin/perl -e 'chomp($$_ = `pwd`); while ($$_ ne "" && ! -e "$$_/$(1)") { m:(.*)/[^/]+/??:; $$_ = $$1; } print;')
-
-ifndef BSG_MANYCORE_DIR
-  export BSG_MANYCORE_DIR := $(call find-dir-with,.BSG_MANYCORE_ROOT)
-endif
+.DEFAULT_GOAL = all
 
+export BSG_MANYCORE_DIR := $(shell git rev-parse --show-toplevel)
```

Comment:
I like this. But what happens if bsg_manycore is inside of bsg_bladerunner?

Reply:
Shows the toplevel dir name of the repo this Makefile is running in... so still would be correct!